open_samus_returns_rando/lua_editor.py

```diff
@@ -92,7 +92,7 @@ def get_script_class(self, pickup: dict, actordef_name: str = "") -> str:
             "name": class_name,
             "resources": resources,
             "parent": parent,
-            "caption": lua_util.wrap_string(pickup["caption"]),
+            "caption": lua_util.wrap_string(pickup["caption"].replace("\n", " ")),
```

Comment:
Does this mean the cptions will all be one line and weird?

Reply:
Does the game support multiline captions? 🙃
If yes, I maybe should escape the \n instead of removing it

Reply:
Alright. Adressed